journal.py

```diff
@@ -194,10 +254,13 @@ def main():
     config.add_static_view('static', os.path.join(HERE, 'static'))
     config.add_route('home', '/')
     config.add_route('add', '/add')
+    config.add_route('ajax_add', '/ajax_add')
+    config.add_route('added', '/added/{title}')
```

Comment:
now I see why you have the `title` as a lookup param for your `last` method.  But I'm still confused about why it is required to do things this way.  I'd love an explanation.